packages/summer-earn-protocol-subgraph/subgraph.yaml

```
@@ -63,15 +58,15 @@ dataSources:
         - handler: handleInterval
           filter:
             kind: polling
-            every: 30 
+            every: 240 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)